exporter/opentelemetry-exporter-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -440,8 +455,65 @@ def test_export_json_max_tag_length(self):
         _, kwargs = mock_post.call_args  # pylint: disable=E0633
 
         tags = json.loads(kwargs["data"])[0]["tags"]
-        self.assertEqual(len(tags["k1"]), 128)
-        self.assertEqual(len(tags["k2"]), 50)
+
+        self.assertEqual(len(tags["string1"]), 128)
+        self.assertEqual(len(tags["string2"]), 50)
+        self.assertEqual(
+            tags["list1"],
+            '["a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a", "a"]',
```

Comment:
I couldn't find anything regarding this in the spec, but can we compress this down by removing whitespace in the `json.dumps()` call?

Reply:
Couldn't find anything either. It will result in slightly less readability on the Zipkin side but feels like a good tradeoff so implemented the compression.